### ISSUE TITLE
UserWarning: None of the inputs have requires_grad=True. Gradients will be None   warnings.warn(

### ISSUE THREAD

**Issue by LegacySLP95** [bmaltais/kohya_ss#3365]
Received the following warning out of the blue while training a recent LoRA: UserWarning: None of the inputs have requires_grad=True. Gradients will be None
  warnings.warn(

Not sure what would have caused this since I always use the same training parameters and haven't made any changes. I don't see anywhere in the GUI to address this issue other than gradient checkpointing in the advanced settings.  It is however already checked. Any advice?
